sale_manual_delivery/models/procurement.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    carrier_id = fields.Many2one(
```

Comment:
@elicoidal Thanks for the feedbacks. One of the goals of this manual delivery, is to separate the delivery from the sale. The sale order is made once in the beginning of the project. Then parts are shipped later and be specified with another carrier.